app/controllers/mediated_pages_controller.rb

```diff
@@ -27,11 +27,12 @@ def validate_mediated_pageable
     fail UnmediateableItemError unless @mediated_page.mediateable?
   end
 
+  # this should not be page param (needs test)
```

Comment:
Good catch. Thanks! This was actually addressed in 215035ebbfb39c8af2052cfdc520cef2ee1a2417 but I neglected to remove the comment.  Just took that out and pushed the branch back up.